goss-testing/scripts/check_bios_firmware_versions.sh

```diff
@@ -347,7 +330,9 @@ does_bios_meet_req() {
     case "$bios_vers" in
       C17) echo "=====> $bmc: BIOS: $bios_vers OK" ;;
       C21) echo "=====> $bmc: BIOS: $bios_vers OK" ;;
-      *) echo "=====> $bmc: BIOS: $bios_vers Unsupported (expected C17 or C21)"
+      C27) echo "=====> $bmc: BIOS: $bios_vers OK" ;;
+      C33) echo "=====> $bmc: BIOS: $bios_vers OK" ;;
```

Comment:
should we be enforcing c33?  meaning, shouldn't we be calling older versions unsupported?

Reply:
My concern with that is related to timing.  This test runs while still in PIT mode and prior to FAS/HFP updates.  I'm afraid this test would fail for every fresh install and upgrade.

Reply:
Shandy was an outlier in that they upgraded to C33 on some nodes while still running raspberry.